mmv1/third_party/terraform/transport/config.go.erb

```diff
@@ -1159,12 +1160,25 @@ func (c *Config) GetCredentials(clientScopes []string, initialCredentialsOnly bo
 		}, nil
 	}
 
+	// UniverseDomain is set by the credential file's "universe_domain" field
```

Comment:
This appears in both service account creds and BYOID?

Reply:
(not new to this PR, just double-checking)

Reply:
Both headful and headless BYOID logins I tested ended up being JSON credential files that contained a `universe_domain`, so yes.